lib/iris/tests/unit/analysis/maths/test__output_dtype.py

```diff
@@ -42,12 +42,17 @@ def setUp(self):
                                 operator.sub,
                                 operator.mul,
                                 operator.pow,
-                                operator.div,
+                                operator.floordiv,
```

Comment:
This is a list of operators which result in the same dtype as their arguments. `truediv` is in a list below.